airbyte-integrations/connectors/source-mysql/src/main/java/io/airbyte/integrations/source/mysql/MySqlSource.java

```diff
@@ -243,7 +243,8 @@ public JsonNode toDatabaseConfig(final JsonNode config) {
     // When using this approach MySql creates a temporary table which may have some effect on db
     // performance.
     jdbcUrl.append("?useCursorFetch=true");
-    jdbcUrl.append("&zeroDateTimeBehavior=convertToNull");
+    // What should happen when the driver encounters DATETIME values that are composed entirely of zeros https://dev.mysql.com/doc/connector-j/8.1/en/connector-j-connp-props-datetime-types-processing.html#cj-conn-prop_zeroDateTimeBehavior
+    jdbcUrl.append("&zeroDateTimeBehavior=CONVERT_TO_NULL");
```

Comment:
`convertToNull` used to be an acceptable value in the past

Reply:
it seems this is changed (or was changed at some point) https://dev.mysql.com/doc/connector-j/8.1/en/connector-j-connp-props-datetime-types-processing.html#cj-conn-prop_zeroDateTimeBehavior